### ISSUE TITLE
Web UI: Require login before users can access the new dashboard

### ISSUE THREAD

**Issue by Tsu-ba-me** [ClusterLabs/anvil#106]
Endpoints for the new dashboard in `/var/www/cgi-bin` need to be protected via authentication. There are at least 2 tasks involved:
- [x] Configure Apache to use `/cgi-bin/striker` as the first page (instead of `/index.html`); this will for the user to land on the login page.
- [x] Modify `/cgi-bin/striker` to redirect to `/index.html` after the user has logged in.

**Comment by Tsu-ba-me** [comment]
A reminder to include the `Header` line for the `cgi-bin` directory in `/etc/httpd/conf/httpd.conf`:
```
<Directory "/var/www/cgi-bin">
  ...

  Header set Access-Control-Allow-Origin "*" 
</Directory>
```

This is required for the client to periodically fetch data from the striker to update the new dashboard.

**Comment by digimer** [comment]
Change from:
```
<IfModule dir_module>
    DirectoryIndex index.html
</IfModule>

```
to:
```
<IfModule dir_module>
    DirectoryIndex cgi-bin/striker
</IfModule>

```

**Comment by Tsu-ba-me** [comment]
The Apache config related changes are implemented in #169
